tracer/operation/operation.go

```diff
@@ -50,15 +50,21 @@ const (
 	GetCommittedStateLclsID
 	GetNonceID
 	GetStateID
+	GetTransientStateID
 	GetStateLccsID
+	GetTransientStateLccsID
 	GetStateLcID
+	GetTransientStateLcID
 	GetStateLclsID
+	GetTransientStateLclsID
 	HasSuicidedID
 	RevertToSnapshotID
 	SetCodeID
 	SetNonceID
 	SetStateID
+	SetTransientStateID
 	SetStateLclsID
+	SetTransientStateLclsID
```

Comment:
Operation Profiler uses the list order to assign OpId. In a single run, adding new Op mid-list might not matter, but it will be confusing to have different OpId means different Op across multiple runs. (Ultimately, we also record OpName so we can resolve any confusion.)

Adding new op only at the end of the list would be the most convenient approach, imho. Another option is to modify how Op Profiler assign OpId.

Reply:
Thanks for the heads up. I will add it to the end.